Dockerfile

```diff
@@ -18,6 +18,7 @@ _JAVA_OPTIONS="-XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap
 WORKDIR /tmp
 
 RUN echo "http://dl-cdn.alpinelinux.org/alpine/latest-stable/community" >> /etc/apk/repositories \
+ && echo "http://dl-cdn.alpinelinux.org/alpine/edge/main" >> /etc/apk/repositories \
```

Comment:
Was getting errors where some packages could not be found. Added `edge` repo fixed that